cmd/up/up.go

```diff
@@ -295,6 +297,22 @@ func Up() *cobra.Command {
 				up.Dev.Autocreate = true
 			}
 
+			// only if the context is an okteto one, we should verify if the namespace has to be woken up
+			if okteto.Context().IsOkteto {
+				k8sClient, _, err := okteto.NewK8sClientProvider().Provide(okteto.Context().Cfg)
+				if err != nil {
+					return err
+				}
+				okClient, err := okteto.NewOktetoClient()
+				if err != nil {
+					return err
+				}
+				if err := wakeNamespaceIfApplies(ctx, up.Dev.Namespace, k8sClient, okClient); err != nil {
+					// If there is an error waking up namespace, we don't want to fail the up command
```

Comment:
if the up command is not directly affected by a wake-up error, could we run `wakeNamespaceIfApplies` in a goroutine?

Reply:
Yes, I'll do it in a goroutine to avoid to impact on the command performance when everything is up